docs/requirements.txt

```diff
@@ -1,14 +1,9 @@
 bioframe
-cooler
 click>=7.0
-cython
 ipython
 nbsphinx
 numpy
```

Comment:
```suggestion
numpy
multiprocess
```

Reply:
Not `numpy` because there's a np.array literal in `plotting` right now that needs numpy to work.

Reply:
I don't know how to request `+multiprocess` alone - this was meant to be 
```
-numpy
+numpy
+multiprocess
```

Reply:
there are just tooo many buttons on github ...

Reply:
Yeah, I got confused with mock and requirements... I meant to keep numpy too.